pkg/operator/ceph/cluster/mgr/dashboard.go

```diff
@@ -104,7 +100,13 @@ func (c *Cluster) configureDashboardModules() error {
 	}
 	if hasChanged {
 		logger.Info("dashboard config has changed. restarting the dashboard module")
-		return c.restartDashboard()
+		if err := c.restartDashboard(); err != nil {
```

Comment:
Don't we need to restart the dashboard after the ssl cert is generated? Now we won't restart the dashboard if the ssl setting changes, won't that be a problem? Perhaps we need another dashboard restart inside `initializeSecureDashboard()` in that case.

Reply:
https://github.com/rook/rook/blob/473d8f89cfebda07316e3f178e7f2930ff05942d/pkg/operator/ceph/cluster/mgr/dashboard.go#L91 The response is always false,  if the configure of the dashboard  not change, it will not restart under previous reality. we had tested with this case?

Reply:
other question is:  after the ssl cert is generated, need to restart dashboard? we had tested with this case? may need to test the case.

Reply:
@travisn please take a look at it